### ISSUE TITLE
QT notifications are now always on the monitor that telegram is on

### ISSUE THREAD

**Issue by PandorasFox** [telegramdesktop/tdesktop#2510]
### Steps to reproduce
1.  Have two monitors
2. Have telegram on monitor A
3. Have monitor B focused and receive a notification
### Expected behaviour

The notification should show on the active monitor, as they did before
### Actual behaviour

The notifications now only arrive on the same monitor as Telegram
### Configuration

**Operating system:**
Arch Linux x64

**Version of Telegram Desktop:**
10.12
### Logs

I don't think any are needed

You can type `debugmode` in settings and then see ~/.TelegramDesktop/DebugLogs/log_...txt for log files.
Type `debugmode` in settings again to disable logs.

**Comment by flippinger** [comment]
same on Windows 10 1607.

The notifications now appear on the monitor telegram is on. Previously they appeared on the main monitor, no matter where Telegram was, which was way better.

Maybe you could add an option to the notification positioning where you can set the monitor?

**Comment by chris13524** [comment]
I'd like to see this as well. It's hard to notice a notification when it's almost out of my peripheral vision.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 363 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by kosgdk** [comment]
Please, reopen this issue.

**Comment by PandorasFox** [comment]
Yeah, I never noticed this was closed. I'd rather it get reopened since it's still an issue and could still be improved somewhat.

**Comment by Mokori** [comment]
Are there any chances to change it?

**Comment by Aokromes** [comment]
#3681 ?
